app/src/main/java/com/example/makeiteven2/intefaces/IEndDialogBtnClicked.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.makeiteven2.intefaces
+
+import android.view.View
+
+interface IEndDialogBtnClicked {
```

Comment:
change the name to Listener in the end - more readable

Reply:
done